templates/index.html

```diff
@@ -6,13 +6,14 @@
       <div class="row mt-3">
         <div class="col-md-4 mx-auto">
           <div class="form-group">
+            <label class="heading" for="bookmark">Fields with * after it are required<span>
             <label class="heading" id="label" for="email">Email *</label>
             <div class="input-group margin-bottom-sm">
               <span class="input-group-addon"><i class="fa fa-envelope-o fa-fw"></i></span>
-              <input autofocus="" class="form-control" placeholder="Enter email" name="email" title="Email for sending the ISO"
+              <input autofocus="" pattern="https?://.+" class="form-control" placeholder="Enter email" name="email" title="Email for the ISO"
```

Comment:
This pattern is for url, why have added it in the email,  it already has a pattern

Reply:
I have misplaced it, sorry!